psi4/src/psi4/libqt/timer.cc

```diff
@@ -1017,9 +1017,9 @@ void timer_done() {
     printer->Printf("Host: %s\n", host);
     free(host);
     printer->Printf("\n");
-    printer->Printf("Timers On : %s", ctime(&timer_start));
+    printer->Printf("Timers On : %s", ctime(&timer_start)); // lgtm[cpp/potentially-dangerous-function] 
```

Comment:
In any idea how this is potentially dangerous?

Reply:
LGTM flagged this because `ctime` is not thread safe. Their suggestion was to use `ctime_r`,  but that failed for the windows builds.

Reply:
Interesting, if you have a moment could you add a comment to that effect in the code base?

Reply:
Sure thing